hazelcast/src/test/java/com/hazelcast/internal/util/futures/ChainingFutureTest.java

```diff
@@ -188,6 +189,16 @@ private InternalCompletableFuture<Object> newFuture() {
         return InternalCompletableFuture.withExecutor(executor);
     }
 
+    @NotNull
```

Comment:
use `javax.annotation.Nonnull` instead of the jetbrains annotation.

This rule should have been enforced by [checkstyle](https://github.com/hazelcast/hazelcast/blob/7c3894b1474dba56648cf10ad8441615486fcdea/checkstyle/checkstyle.xml#L157-L159), but maybe it is only applied on production code?